README.md

```diff
@@ -69,14 +69,14 @@ Getting started
 
 ### Building images
 
-To build images, define an imagebuilding playbook (see example.yml for an example) or
+To build images, define an imagebuilding script (see example.sh for an example) or
 use the default one. For example, to build the default images:
 
 ```
-sudo ansible-playbook -i hosts all.yml
+$ sudo ./on-imagebuilder/build_all.sh
 ```
 
-**All ansible runs must be done on the host machine that is building the images.** This is because we use
+**All builds runs must be done on the host machine that is building the images.** This is because we use
```

Comment:
All build runs must be executed locally on the machine that is building the images. ?